client/src/driver/web_socket.js

```diff
@@ -0,0 +1,377 @@
+import { Observable as O } from '../rxjs'
```

Comment:
this driver handshakes with the node (Bolt-8) and returns a websocket object for communication.